packages/loopring_v3/contracts/impl/libexchange/ExchangeGenesis.sol

```diff
@@ -29,12 +35,12 @@ library ExchangeGenesis
         bytes32 _domainSeperator
         )
         external
+        onlyWhenUninitialized(S)
```

Comment:
Already checked in ExchangeV3.initialize, so probably a bit overkill to do it here as well? (maybe even a bit overboard to have a dedicated `ExchangeGenesis` library contract just for this initialization function...)

Reply:
Agreed, reverted.